tools/ansible/docs/ocp-4-4-with-windows-server.md

```diff
@@ -0,0 +1,419 @@
+# OpenShift Container Platform 4.4 with Microsoft Windows Server 2019/1809 Nodes
+
+**Developer Preview**
+
+## Pre-requisites
+
+The document is tested against a Fedora release 31 Linux host for running all
+the Linux commands.
+
+Please note that a command preceded by > is to be run in a PowerShell window on
+a Windows instance, and a command preceded by $ is to be run on a Linux console
+(localhost).
+
+### Installation of prerequisite packages 
+
+Python3 and pip are required to follow this guide. To check if python3 is
+installed on the system, run the following command:
+
+```sh
+$ python3 --version
+```
+
+To install python3, run the following command:
+
+```sh
+$ sudo dnf install python3
+```
+
+To check if pip is installed on the system, run the following command:
+
+```sh
+$ python3 -m pip --version
+pip 19.0.3 from /usr/lib/python3.7/site-packages/pip (python 3.7)
+```
+
+To install pip refer [pip installation](https://pip.pypa.io/en/stable/installing/)
+
+Install jq library to parse json files:
+
+```sh
+$ sudo dnf install jq
+```
+
+Install git to clone github repository:
+
+```sh
+$ sudo dnf install git
+```
+
+## Bring up the OpenShift cluster with ovn-kubernetes
+
+Download the dev-preview 4.4.0-rc.3
+[OpenShift Installer and Client](https://mirror.openshift.com/pub/openshift-v4/x86_64/clients/ocp/4.4.0-rc.3/)
```

Comment:
Please point to [OpenShift 4.4.4](https://mirror.openshift.com/pub/openshift-v4/clients/ocp/4.4.4/) here and in the other docs.